js.go

```diff
@@ -981,11 +986,21 @@ type ConsumerConfig struct {
 	HeadersOnly     bool            `json:"headers_only,omitempty"`
 
 	// Pull based options.
-	MaxRequestBatch   int           `json:"max_batch,omitempty"`
-	MaxRequestExpires time.Duration `json:"max_expires,omitempty"`
+	MaxRequestBatch    int           `json:"max_batch,omitempty"`
+	MaxRequestExpires  time.Duration `json:"max_expires,omitempty"`
+	MaxRequestMaxBytes int           `json:"max_bytes,omitempty"`
```

Comment:
Wonder if we should leave this one out?

Reply:
True. Without ability to send a pull request with max bytes, there is little value. I can remove.

Reply:
Done. Sorry for the "git push force", it is because I wanted to remove the field from the git comment itself.